rotkehlchen/globaldb/migrations/migration1.py

```diff
@@ -43,22 +43,10 @@
 
 def globaldb_data_migration_1(conn: 'DBConnection') -> None:
     """Introduced at 1.27.1
-    - Adds the ilk registry contract for ethereum mainnet
     - Removes old setting last_assets_json_version (if existing)
     - Adds a makerdao vault types cache
     """
     with conn.write_ctx() as write_cursor:
-        # Write the ilk registry contract
-        write_cursor.execute(
```

Comment:
Why was this removed? Is it because it's already in the DB and the migration, does not make sense?

Reply:
Exactly